pkg/rbd/nodeserver.go

```diff
@@ -297,7 +298,7 @@ func getLegacyVolumeName(mountPath string) (string, error) {
 
 func (ns *NodeServer) mountVolumeToStagePath(ctx context.Context, req *csi.NodeStageVolumeRequest, stagingPath, devicePath string) error {
 	fsType := req.GetVolumeCapability().GetMount().GetFsType()
-	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: mount.NewOsExec()}
+	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: utilexec.New()}
```

Comment:
These kind of changes could have been mentioned in the commit message. It is not only the Gopkg.toml update and vendor/, but something more.